server/src/controllers/projectsController.ts

```
@@ -1,18 +1,22 @@
 import { Request, Response } from 'express';
-import { projects } from '../data/projects';
+import { PrismaClient, Prisma } from '@prisma/client';
 
-export const getProjects = (req: Request, res: Response) => {
-  const search = req.query.search?.toString().toLowerCase();
+const prisma = new PrismaClient();
 
-  if (!search) {
-    return res.json(projects);
+export const getProjects = async (req: Request, res: Response) => {
+  const search = req.query.search?.toString().toLowerCase();
+  try {
+    const where: Prisma.projectsWhereInput | undefined = search
+      ? {
+          OR: [
+            { title: { contains: search, mode: 'insensitive' } },
+            { description: { contains: search, mode: 'insensitive' } },
+          ],
+        }
+      : undefined;
+    const projects = await prisma.projects.findMany({ where });
+    res.json(projects);
+  } catch (error) {
+    console.error(error);
```

we probably should return an error as everywhere